Parsers/ASimWebSession/Parsers/ASimWebSessionSquidProxy.yaml

```diff
@@ -29,7 +29,7 @@ ParserQuery: |
   SquidProxy_CL | where not(disabled)
     | extend AccessRawLog = extract_all(@"^(\d+\.\d+)\s+(\d+)\s(\S+)\s([A-Z_]+)\/(\d+)\s(\d+)\s([A-Z]+)\s(\S+)\s(\S+)\s([A-Z_]+)\/(\S+)\s(\S+)",dynamic([1,2,3,4,5,6,7,8,9,10,11,12]),RawData)[0]
     | project-rename
-      Dvc = Computer
+      Dvc = 'Squid Proxy'
```

Comment:
@KostaS10 Can you use this instead, see if it works fine for you. This should handle both when Computer exists or not.

| extend Dvc = iff(isnotempty(column_ifexists("Computer", "")), column_ifexists("Computer", ""), "Squid Proxy")

Reply:
Updated to reflect your code, @vakohl